src/MarkdownTextInput.web.tsx

```diff
@@ -236,6 +236,35 @@ const MarkdownTextInput = React.forwardRef<MarkdownTextInput, MarkdownTextInputP
       [parser, parseText, processedMarkdownStyle],
     );
 
+    const format = useCallback(
+      (target: MarkdownTextInputElement, parsedText: string, cursorPosition: number, formatType: string): ParseTextResult => {
+        if (!contentSelection.current) {
+          return {
+            text: '',
+            cursorPosition: 0,
+          };
+        }
+        let markdown;
+        switch (formatType) {
+          case 'formatBold':
+            markdown = '*';
+            break;
+          case 'formatItalic':
+            markdown = '_';
+            break;
+          default:
+            markdown = '';
```

Comment:
If `formatType` is other than `formatBold` or `formatItalic`, then we'll still call the code below. Is it possible to early-return here instead of running the code below with `markdown = ''`?